### DISCUSSION TITLE
Umi 4 的API是否要从 import from 'umi' 改为 import from '@umijs/max' ?

### DISCUSSION THREAD

**Discussion by xliez** [umijs/umi#10641] [Q&A] [answered]
使用 @umijs/max 4.0.53 创建的项目
发现 `import { history, useModel } from 'umi'` 的话，TS会提示 `Cannot find module 'umi' or its corresponding type declarations.`

**Comment by fz6m** [comment, marked as answer]
max 项目都需要从 `@umijs/max` 导入，umi 项目都需要从 `umi` 导入。

**Reply by xliez** [reply]
官方文档是没更新吗，看了《升级到Umi4》《QA》都没有提到相关说明
在 max 的文档下面的示例也是使用的 import from 'umi' [request](https://umijs.org/docs/max/request)
